en/config_mc/pid_tuning_guide_multicopter.md

```diff
@@ -206,23 +206,43 @@ The following parameters can also be adjusted. These determine the maximum rotat
 ### Thrust Curve
 
 The tuning above optimises performance around the hover throttle.
-But it can be that you start to see oscillations when going towards full throttle.
+But you may start to see oscillations when going towards full throttle.
 
-To counteract that adjust the **thrust curve** with the [THR_MDL_FAC](../advanced_config/parameter_reference.md#THR_MDL_FAC) parameter.
+To counteract that, adjust the **thrust curve** with the [THR_MDL_FAC](../advanced_config/parameter_reference.md#THR_MDL_FAC) parameter.
 
 > **Note** The rate controller might need to be re-tuned if you change this parameter.
 
 The mapping from motor control signals (e.g. PWM) to expected thrust is linear by default — setting `THR_MDL_FAC` to 1 makes it quadratic.
 Values in between use a linear interpolation of the two. Typical values are between 0.3 and 0.5.
 
-If you have a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer) (or can otherwise _measure_ thrust),
-you can determine the relationship between the PWM control signal and the motor's actual thrust, and fit a function to the data.
-[This Notebook][THR_MDL_FAC_Calculation] shows how the thrust model factor `THR_MDL_FAC` may be calculated from previously measured thrust data.
+If you have a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer) (or can otherwise _measure_ thrust and PWM commands simultaneously), you can determine the relationship between the PWM control signal and the motor's actual thrust, and fit a function to the data.
```

Comment:
```suggestion
If you have a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer) (or can otherwise _measure_ thrust and PWM commands simultaneously), you can determine the relationship between the motor control signal and the motor's actual thrust, and fit a function to the data. The motor command in PX4 called `actuator_output` can be PWM, Dshot, UAVCAN commands for the respective ESCs in  use. 
```

Reply:
Sounds good to me